entities/Order.java

```diff
@@ -0,0 +1,59 @@
+package entities;
+
+import constants.Status;
+
+import java.time.Instant;
+import java.util.Date;
+import java.util.List;
+
+public class Order extends Base {
+    //Attributes
+
+    private String customerName;
+    private List<Product> items;
+    private Double totalAmount;
+
+    //Constructors
+
+    public Order() {
+    }
+
+    public Order(Long id, Instant createdDate, Instant updatedDate, Long createdBy, Long updatedBy, Status status, String customerName, List<Product> items, Double totalAmount, Date orderDate) {
+        super(id, createdDate, updatedDate, createdBy, updatedBy, status);
+        this.customerName = customerName;
+        this.items = items;
+        this.totalAmount = totalAmount;
+    }
+
+    //Methods
+
+    public String getCustomerName() {
+        return customerName;
+    }
+
+    public void setCustomerName(String customerName) {
+        this.customerName = customerName;
+    }
```

Comment:
You might need more information about the customer, so either remove this or refer to a customer entity instead

Reply:
Referred to customer entity, It will be the next one to be created.